network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/news_you_can_use.html

```diff
@@ -56,9 +60,13 @@ <h3 class="mb-2 pb-1"><a href="{{first.url}}">{{ first.title }}</a></h3>
         <div class="bg-white">
           {% image item.blog.get_meta_image fill-700x394 class="embed-responsive-item" %}
           <div class="p-4">
-            {% with category=item.blog.category.first %}
-            <a class="h6-heading d-block mb-1" href="/blog/category/{{ category.slug }}">{{ category }}</a>
-            {% endwith %}
+            {% if item.blog.category.count %}
+              {% with category=item.blog.category.first %}
+                <a class="h6-heading d-block mb-1" href="/blog/category/{{ category.slug }}">{{ category }}</a>
+              {% endwith %}
+            {% else %}
+                <div class="h6-heading d-md-block d-none mb-2">&nbsp;</div>
```

Comment:
For desktop/tablet, I think the empty card contents is sitting slightly lower so I believe in this (empty state) area it should have `mb-1` instead of `mb-2` to match with the cards with categories. 

Mobile looks good though.

Reply:
Correct, good catch! Will fix and merge. Thanks @sabrinang